docs/main_text.md

```diff
@@ -437,7 +437,17 @@ We then quantile normalize all samples as described above.
 
 We've made our analyses underlying processing choices and exploring test compendia available at our <a href = "https://github.com/AlexsLemonade/compendium-processing" target = "blank">`compendium-processing`</a> repository.
 
-### Download Folder
+### Collapsing by genus
+
+Microarray platforms are generally designed to assay samples from a specific species.
+In some cases, publicly available data surveyed by refine.bio may include samples where the microarray platform used was not specifically designed for the species as described (e.g., samples labeled _Bos indicus_ will be run on _Bos taurus_ microarrays or mouse crosses that are not labeled _Mus musculus_ will be run on _Mus musculus_ microarrays). 
```

Comment:
Should we say `were run` instead of `will be run` to make it more obvious that these runs were already done by external researchers?

Reply:
How about `are run` ?